Production/govuk_ios/ViewControllers/SettingsViewController.swift

```diff
@@ -21,5 +23,61 @@ class SettingsViewController: BaseViewController,
         super.viewDidLoad()
 
         view.backgroundColor = UIColor.govUK.fills.surfaceBackground
+
+        title = viewModel.title
+        navigationController?.setNavigationBarHidden(false, animated: false)
+        navigationController?.navigationBar.prefersLargeTitles = true
+
+        scrollview.translatesAutoresizingMaskIntoConstraints = false
+        view.addSubview(scrollview)
+
+        configureUI()
+        configureConstraints()
+    }
+
+    override func viewDidAppear(_ animated: Bool) {
+        super.viewDidAppear(animated)
+
+        analyticsService.track(screen: self)
```

Comment:
As this is a BaseViewController, it is tracked automagically when it conforms to `TrackableScreen` so shouldn't need this.